uber/site_sections/preregistration.py

```diff
@@ -523,32 +539,21 @@ def event(self, session, id, slug, register=None):
     def credit_card_retry(self):
         return {}
 
```

Comment:
This is another case of pulling config-based if statements out of the module level and into functions.